src/connectors/infinite-hits/connectInfiniteHits.js

```diff
@@ -68,7 +68,11 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
   checkRendering(renderFn, withUsage());
 
   return (widgetParams = {}) => {
-    const { escapeHTML = true, transformItems = items => items } = widgetParams;
+    const {
+      escapeHTML = true,
+      transformItems = items => items,
+      showPrevious,
```

Comment:
We should now provide the default value at this level rather than inside the widget.